lib/Basics/errors.dat

```diff
@@ -68,6 +68,7 @@ ERROR_ARANGO_DATAFILE_UNREADABLE,1006,"datafile unreadable","Internal error that
 ERROR_ARANGO_DATAFILE_EMPTY,1007,"datafile empty","Internal error that will be raised when a datafile is empty."
 ERROR_ARANGO_RECOVERY,1008,"logfile recovery error","Will be raised when an error occurred during WAL log file recovery."
 ERROR_ARANGO_DATAFILE_STATISTICS_NOT_FOUND,1009,"datafile statistics not found","Will be raised when a required datafile statistics object was not found."
+ERROR_ARANGO_ICU_ERROR,1010,","icu error: %s","will be raised if icu operations failed"
```

Comment:
please let's not have quotes inside the error messages.
Why not use `ICU error: %s` for the message text?

Reply:
Additionally, we should not put there error into category `10xx`, because the description for this error category is:
```
################################################################################
## Internal ArangoDB storage errors
## For errors that occur because of a programming error.
################################################################################
```